### ISSUE TITLE
Hyperparameter setting about the 300W training

### ISSUE THREAD

**Issue by FunkyKoki** [zongjuede/adaptive-wing-loss#2]
Hi there, thank you so much for sharing the AWing loss code.
Can you share your 300W training settings, such as optimizer setting, batch_size, learning rate, data augmentation?

**Comment by zongjuede** [comment]
Hyper parameters are decided by network architecture, dataset, task type and so on.
The commonest hyper parameters I used for facial landmark training (including coordinate regression and heatmap regression) are :
Optimizer: Adam
Lr_scheduler: ReduceLROnPlateau
Batch size: 8 (It seems that small batch size always work better.)
Learning rate: 0.001 (1e-3)
Data augmentation: random rotation, random translation, random flipping, random occlusion and so on.

**Comment by FunkyKoki** [comment]
Thank you for your answer. That helps a lot.
